code/02_calculate-measures.R

```diff
@@ -198,6 +200,7 @@ pds %<>%
 # write final data
 pds %>% 
 write_file(path = get_mi_data_path("final_data", ext = "rds", test_output = test_output))
+0 # this zero stops script from running IF write_file is overwriting an existing file, re-run the section without this line and enter 1 in the console, when prompted, to overwrite file.
```

Comment:
One question i had was how do we apply this to the mi report itself? we wrap that in the `render` function to create the mi report which wont use the `write_file` function. I wonder if there is a way to add another check for the mi report and also the publication outputs?

Reply:
Good question. I hadn't considered the mi itself or the publication outputs. I will look into it.

Reply:
Thanks, I wasnt too sure about what code to suggest because we were wrapping it in `render` i dont think it would work since we created the `write_file` function and it works differently